panel/lxqtpanel.cpp

```diff
@@ -160,19 +173,23 @@ LXQtPanel::LXQtPanel(const QString &configGroup, LXQt::Settings *settings, QWidg
     setLayout(lav);
     this->layout()->addWidget(LXQtPanelWidget);
 
+    //Create, initialize and connect the LXQtPanelLayout.
```

Comment:
It seems to me that self-explanatory is somewhat dependent on how much you have already understood the code. When I'm reading code, I'd rather have one comment too much than one comment too few. That's why I've added these comments even if they did not seem completely necessary but this could be just my personal opinion. Furthermore, I think that these comments can group the code into logical units.

If you do not like this one, for example, I guess you'd like me to remove most of the comments, right?
Do you make any difference between header and cpp files? In the header file, I have tried to explain how the different parts fit together and build up the class, so I was very generous with comments. Was that too much?